src/edit/DistortableCollection.Edit.js

```diff
@@ -303,28 +303,27 @@ L.DistortableCollection.Edit = L.Handler.extend({
         statusUrl = opts.statusUrl + data;
         // repeatedly fetch the status.json
         self.updateInterval = setInterval(function intervalUpdater() {
-          $.ajax(statusUrl + '?' + Date.now(), {// bust cache with timestamp
-            type: 'GET',
-            crossDomain: true,
-          }).done(function(data) {
-            opts.updater(data);
-          });
+          var request = new XMLHttpRequest();
+          request.onload = function(event) {
```

Comment:
`.done` runs only on success, so to mirror that via xhr I think we should add some a conditional inside `onload` like `if (request.status === 200) { //... }`

Reply:
Sorry for the long time to respond, things are shaky.
I missed the validation sorry, I have added the validation.

Reply:
no problem! I will take a look again now